tests/scripts/thread-cert/Cert_5_1_01_RouterAttach.py

```diff
@@ -61,6 +70,8 @@ def test(self):
         self.simulator.go(7)
         self.assertEqual(self.nodes[ROUTER].get_state(), 'router')
 
+        self.collect_rloc16s()
+
```

Comment:
should the packet verification code with deprecated method below be removed?

Reply:
I thinks these duplicate verification should be removed

Reply:
FYI: if we remove the old verification code, we can set `USE_MESSAGE_FACTORY = False` for the test case to be a little bit more efficient. @FiveDimensions

Reply:
added